.github/workflows/stale.yml

```
@@ -0,0 +1,37 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '35 3 * * *'
+
+jobs:
+  stale:
+
+    runs-on: shopify-ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
```

I believe actions in public repos need to use public runners